README.md

```diff
@@ -1,49 +1,41 @@
 # Coverage Checker
 
-이 스크립트는 신규 코드에 대한 코드 커버리지를 측정하여 프로젝트 기준에 충족하는지를 판별하고  
-해당 pr에서 변경된 소스코드의 코드 커버리지가 어떻게 되는지 풀 리퀘스트에 피드백을 해줍니다.
+CoverChecker check test coverage of new line of code. If you create new pull request then CoverChecker feedback to  how much test covered new lines.
 
-**코드 커버리지**는 작성한 코드에 대해 얼마나 테스트코드를 통해 검증되었는지 확인하는 지표입니다.
-제대로 작성된 테스트코드는 릴리즈 이전 소스코드의 버그를 미리 탐지하고 
-그로 인해 발생할 수 있는 장애를 미연에 방지할 수 있습니다.
+Code coverage is describe one of code quality. Good test code find bug before release also prevent service disable.
 
-> 최근 스프링의 컴포넌트 중 하나인 [restdoc](https://www.baeldung.com/spring-rest-docs)의 경우 작성한 
-테스트 코드를 참고하여 자동으로 api 문서를 만들어주기도 합니다.
+> [Spring REST doc](https://spring.io/projects/spring-restdocs) is one of spring component. it makes api document from test code.
 
-이런 이유로 많은 팀에서 테스트 코드를 적용하기 위해 노력하고 있지만 
-몇 천 줄은 가뿐히 넘는 기존 코드상에 테스트 코드를 전부 적용하기란 쉽지 않습니다.
-이 스크립트를 통해 새로 작성 / 수정하는 코드들에 조금씩 커버리지를 늘려 
-프로젝트의 전반적인 커버리지도 올리고 각 팀원의 프로그래밍 습관도 개선 해보세요.
+Therefore many developer wants write test code and improve code coverage. But old legacy code and project release date is not allow write them. CoverChecker will reduce pressure and get more coverage. It makes your code more durable.
```

Comment:
I think **"Therefore many developers want to write"** is more natural than **"Therefore many developer wants write"**